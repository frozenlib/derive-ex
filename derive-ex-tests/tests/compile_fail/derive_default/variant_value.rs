use derive_ex::derive_ex;

#[derive_ex(Default)]
enum X {
    #[default(10)]
    A,
}
fn main() {}
