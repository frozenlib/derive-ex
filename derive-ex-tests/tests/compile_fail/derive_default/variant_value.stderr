error: `#[default(...)]` on a variant cannot specify a default value
 --> tests/compile_fail/derive_default/variant_value.rs:5:15
  |
5 |     #[default(10)]
  |               ^^
